client/verta/tests/test_versioning/test_dataset.py

```diff
@@ -9,6 +9,24 @@
 import verta.dataset
 
 
+def walk_files(dirpath):
+    for root, _, filenames in os.walk(dirpath):
+        for filename in filenames:
+            yield os.path.join(root, filename)
+
+
+def assert_dirs_match(dirpath1, dirpath2):
```

Comment:
If I'd be a little bit annoying, I could say that `dirpath2` can have more files than `dirpath1`. Not sure if it's worth covering that.

Reply:
Ooooooh how did I miss that??
That's definitely something I'd want covered by this.